### ISSUE TITLE
Smarter tp/fp comments from metasmoke on blacklist request

### ISSUE THREAD

**Issue by rjrudman** [Charcoal-SE/SmokeDetector#441]
See [here](https://github.com/Charcoal-SE/SmokeDetector/pull/440) for an example. 

A user requested that a username should be blacklisted - which prompted the usual metasmoke comment. However - the result only checks for true/false positives in post bodies, rather than the username. 

- [ ] Fix comment
- [x] Fix link in PR

**Comment by Undo1** [comment]
@angussidney Are you doing both sides of this?

**Comment by angussidney** [comment]
@Undo1 Completely forgot about the comment :) I can do the search link (since that can be done smokey side), but not the comment.

**Comment by angussidney** [comment]
MS doesn't seem to be leaving a comment

**Comment by quartata** [comment]
I'm not sure why this isn't working -- SmokeDetector is including the proper metadata, so the only thing I can think of is that the MS webhook has been disconnected (@ArtOfCode-)?

**Comment by ArtOfCode-** [comment]
@Undo1 will need to verify that, since he has the GH secret.

**Comment by Undo1** [comment]
@quartata Dug through the GH hook logs. There's at least two issues here. Attached a couple of backtraces; filenames contain SD PR numbers they happened on.

[metasmoke-backtraces.zip](https://github.com/Charcoal-SE/SmokeDetector/files/1503649/metasmoke-backtraces.zip)

**Comment by ArtOfCode-** [comment]
TL;DR: someone, somewhere, has misunderstood what `String#scan` returns.

**Comment by angussidney** [comment]
Probably my fault, sorry
